### PR TITLE
Staging to merge - staging

### PR DESCRIPTION
**Pull request by prateek-giddh**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/app/inventory/inventory.component.ts [hunk 1]
```diff
@@ -468,6 +467,10 @@ export class InventoryComponent implements OnInit, OnDestroy, AfterViewInit {
                 } else {
                     warehouse = warehouseData.formattedWarehouses[0].uniqueName;
                 }
+                const currentWarehouse = warehouseData.formattedWarehouses.find((data) => data.uniqueName === warehouse || data.value === warehouse);
+                if (currentWarehouse) {
+                    this.warehouseFilter.filter = currentWarehouse.label;
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/inventory/inventory.component.ts:472]: Nullity check warehouseFilter

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl.component.ts [hunk 2]
```diff
@@ -151,7 +151,7 @@ export class PlComponent implements OnInit, AfterViewInit, OnDestroy {
                         cg.childGroups = [];
                         if (['purchasesStockAmount', 'manufacturingExpenses'].includes(f)) {
                             cg.groupName = `+ ${cg.groupName}`;
-                        } else if (f === 'closingInventory') {
+                        } else if (['closingInventory', 'debitNoteStockAmount'].includes(f)) {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/tb-pl-bs/components/pl/pl.component.ts:154]: Rename f

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl.component.ts [hunk 3]
```diff
@@ -132,7 +132,7 @@ export class PlComponent implements OnInit, AfterViewInit, OnDestroy {
                     cogsGrp.accounts = [];
                     cogsGrp.childGroups = [];
 
-                    Object.keys(cogs).filter(f => ['openingInventory', 'closingInventory', 'purchasesStockAmount', 'manufacturingExpenses'].includes(f)).forEach(f => {
+                    Object.keys(cogs).filter(f => ['openingInventory', 'closingInventory', 'purchasesStockAmount', 'manufacturingExpenses', 'debitNoteStockAmount'].includes(f)).forEach(f => {
```
Comment (rohit-walkover) [inline apps/web-giddh/src/app/tb-pl-bs/components/pl/pl.component.ts:135]: rename f

File: apps/web-giddh/src/app/inventory/inventory.component.ts (hunk 1 above)
Comment (prateek-giddh) [reply apps/web-giddh/src/app/inventory/inventory.component.ts:472]: No need as warehouseFilter is viewchild

File: apps/web-giddh/src/app/tb-pl-bs/components/pl/pl.component.ts (hunk 2 above)
Comment (aditya121) [reply apps/web-giddh/src/app/tb-pl-bs/components/pl/pl.component.ts:154]: @rohit-walkover please review the pr https://github.com/Walkover-Web-Solution/Giddh-New-Angular4-App/pull/4684 which contains review changes

**Pull request merged**